loopchain/baseservice/rest_client.py

```diff
@@ -63,6 +64,21 @@ def _init_http_clients(self):
             if version != conf.ApiVersion.v1:
                 self._http_clients[url] = HTTPClient(url)
 
+    async def _fetch_status(self, session: ClientSession, request_uri):
+        endpoint_target = urlparse(request_uri).scheme + "://" + urlparse(request_uri).netloc
```

Comment:
How about...:
```python
from urllib.parse import urljoin
...

endpoint_target = urljoin(request_uri, "/")
```

Reply:
https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin

Reply:
it causes error when endpoint_target is set as urljoin because 443 port is added into dns url.